### ISSUE TITLE
Locate By IP Api service shutdown

### ISSUE THREAD

**Issue by ogadit** [onury/geolocator#64]
Please change the api service for locatebyip as the current one throws off an error saying that it has been shutdown

**Comment by ivanoats** [comment]
@onury would you like a Pull Request for this?

**Comment by onury** [comment]
@ogadit thanks. That's already mentioned in #62.

@ivanoats I actually need help finding a new free + https geo-service to set as default.

**Comment by DarthFly** [comment]
Found one free https://www.geojs.io/docs/v1/endpoints/geo/
Seems to be working correctly.
```
{
                    provider: 'geojs.io',
                    url: 'https://get.geojs.io/v1/ip/geo.js',
                    callbackParam: 'callback',
                    schema: {
                        ip: 'ip',
                        coords: {
                            latitude: 'latitude',
                            longitude: 'longitude'
                        },
                        address: {
                            city: 'city',
                            state: 'state',
                            stateCode: '',
                            postalCode: '',
                            countryCode: 'country_code',
                            country: 'country',
                            region: 'region'
                        }
                    }
                }
```
Also there is another one https://geoip-db.com/ for I was unable to force it with another calback instead of default one `callback(json)`.

**Comment by onury** [comment]
Thansk @DarthFly. Released v2.1.5
